removing references to Macro and Partial view macro (#5979)
umbraco/UmbracoDocs: 13/umbraco-cms/extending/macro-parameter-editors.md

# Macro Parameter Editors

{% hint style="warning" %}

> Macros are deprecated and will be removed in the next version. We recommend using Partial Views or Blocks in Rich Text Editor.

Author: ```suggestion
Macros will be removed in the next version. Consider using Partial Views or Blocks in Rich Text Editor.
```
